src/renderer/SparklineCellRenderer.ts

```diff
@@ -20,10 +20,16 @@ export function line(data: number[]) {
 export default class SparklineCellRenderer implements ICellRendererFactory {
 
   createDOM(col: INumbersColumn & Column): IDOMCellRenderer {
+    const yPos = 1 - col.getMapping().apply(col.getThreshold());
     return {
-      template: `<svg viewBox="0 0 ${col.getDataLength() - 1} 1" preserveAspectRatio="none meet"><path></path></svg>`,
+      template: `<svg viewBox="0 0 ${col.getDataLength() - 1} 1" preserveAspectRatio="none meet"><line x1="0" x2="${col.getDataLength() - 1}" y1="${yPos}" y2="${yPos}"></line><path></path></svg>`,
       update: (n: HTMLElement, d: IDataRow) => {
-        n.firstElementChild!.setAttribute('d', line(col.getNumbers(d.v, d.dataIndex)));
+        const data = col.getNumbers(d.v, d.dataIndex);
```

Comment:
can the `renderMissingValue` function be used?

Reply:
I assume you mean `isMissingValue()`. However, that function doesn't work either, because if some values are missing `d3. line()` cannot create a valid path. I get the following error message:

![image](https://user-images.githubusercontent.com/5851088/32667777-fa324bec-c63b-11e7-9747-b5ee01cf29ab.png)